knowledge-base/apisec-inc/apisec-run-scan/action-security.yml

```diff
@@ -0,0 +1,5 @@
+name: 'apisec run scan action'
```

Comment:
@Devils-Knight please set the name as same in `action.yml` https://github.com/apisec-inc/apisec-run-scan/blob/master/action.yml